docs/integrations/amazon-aws/amazon-elastic-container-service-container-insights-cloudwatch.md

```diff
@@ -0,0 +1,643 @@
+    Add the fields account, region and namespace as shown below.\
+    ![](https://lh6.googleusercontent.com/3B46K8ffMqb_Xewdi9tejJAGNt5ORKfZ9AaQ7Nw8cI1kk8RKE5PsOtdf-wLgBk1Q15Bu11zX-MGuqKN1CXCSr-lI6zrVykcatScyaUFJcvq8wu2POAZ6b0Up41Gl4WhC8oN5WtIqQozojZG4DDt4jGblmCJaTgbFHMXTSbQNJVdupnnoGbQTqp1W_A)
+
+2.  Copy the KinesisLogsRoleARN and KinesisLogsDeliveryStreamARN from the outputs tab of Cloudformation\
+    ![](https://lh3.googleusercontent.com/KtuF8NiCKyQQGoGwgIuITAScCScKB1yP2oCats3bEBYzPVttXglQ6IMPhycmXmbLan39rDcUbeZsreM1EW3H5j9LyebwTGRaMQwvm1GPqg1Dj533F1uyJR_OBoypOxyK5S5T_M8vz_h04LXhqM4FqOlFB-jbRzI-IG1sk7UmeACGHKR1ININ3wSJ-g)
+
+3.  Go to your Cloudwatch -> Log Groups and click on your cloudwatch log group `/aws/ecs/containerinsights/<cluster>/performance`
+    ![](https://lh4.googleusercontent.com/hVccEttjDOWoyXsBT6oNpV-USb6ZdoYzmqYVlzG-RZ_tJzPM3xVY9EQvUlVlCZrzsrPvqeJhlWJgwOoN5My6VepcPihk6yBJ7YkwHK___oqu5nfFMw-e2QcfDlm-QWk7dJjaZ1GHwmIcij51KDxvBn7UVruxUj0OogLN4E3Vx4xys4PJpj47kGShCA)
+
+4.  Click on Create and in opened window fill in the below parameters
+
+    1.  Get the delivery stream name from the arn copied in step 2 and fill in the KinesisLogsDeliverStream  field.
+    2.  Get the role name from the arn copied in step 2 and fill in the role.
+    3.  Specify the filter pattern `**{ $.Type = "Container" || $.Type = "Task" }**`
+    4.  Specify the filter name
+    5.  Test the pattern and click Start streaming\
+        ![](https://lh3.googleusercontent.com/sGGDaTqe2KgIAPCV_TwpFLIxq39qyRH9KgAlvUd7AWqVY8otcxu521_pzVwy4Bl7CfO42xOJH12WuzrIfTjLBP-c0SoLs0xMyusHvmuVZgCY2dBkMTuijMEq2iSzwbmxibNixF3J4PzrAcswO07nOmk7AtazjWBpTlrtgOdsWbWMLRT79srSHpqSAA)
+
+### Collect Application Logs for Amazon ECS
+
+
+Setup the Container logs collection using the steps in following [docs](https://help.sumologic.com/03Send-Data/Collect-from-Other-Data-Sources/AWS_Fargate_log_collection). You can use awsfirelens driver and avoid sending logs to cloudwatch log groups.  Put account, region and namespace fields also while configuring the source.
+
+If your logs are already going to cloudwatch logs groups then you can create a subscription filter to subscribe the log groups to the delivery stream created in the previous step.
+
+![Note](https://help.sumologic.com/@api/deki/files/116/Note.png)Application logs do not contain regions, so you have to configure a new sumo logic source for each region if you want to avoid creating multiple sources, then you will have to put the [X-SUMO-Fields](https://help.sumologic.com/Manage/Fields#x-sumo-fields-http-header) header inside logConfiguration by creating a custom fluent bit image and specify a custom fluent bit configuration. For more information, see, [Create a custom Fluent Bit image](https://help.sumologic.com/03Send-Data/Collect-from-Other-Data-Sources/AWS_Fargate_log_collection#(Optional)_Create_a_custom_Fluent_Bit_image).
```

Comment:
Not how we format notes (see style guide)

Reply:
Corrected all notes

Reply:
This isn't fixed